### PR TITLE
Create Cypress test for viewing answers on PREPARATION

### PR DESCRIPTION
**Pull request by petterwildhagen**

### CODE REVIEW

File: frontend/cypress/integration/view_answers_spec.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            })
+        )
+    before('Load evaluation with answers by two participants', () => {
+        evalWithIndividualAnswers.plant()
+    })
+    it('Participant can only see his/her own answer', () => {
+        cy.visitProgression(Progression.Preparation, evalWithIndividualAnswers.evaluationId, firstParticipant.user, fusionProject1.id)
+        cy.getByDataTestid('view-answers-' + questionOrder).within(() => {
+            cy.getByDataTestid('view-answers').click()
+        })
```
Comment (achaikou) [inline frontend/cypress/integration/view_answers_spec.ts:66]: You already use it twice in this test. Most likely will use more. 
So maybe move to evaluation.ts/EvaluationPage under the name of clickViewAnswers or something?

File: frontend/cypress/integration/view_answers_spec.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        )
+        .addAnswer(
+            new Answer({
+                questionOrder: questionOrder,
+                answeredBy: lead,
+                progression: Progression.Individual,
+                text: leadAnswerText,
+            })
+        )
+    before('Load evaluation with answers by two participants', () => {
```
Comment (achaikou) [inline frontend/cypress/integration/view_answers_spec.ts:59]: Not just two participants? Just with individual answers from all participants?

File: frontend/cypress/integration/view_answers_spec.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        cy.getByDataTestid('view-answers-' + questionOrder).within(() => {
+            cy.getByDataTestid('view-answers').click()
+        })
+        cy.contains(firstParticipantAnswerText).should('be.visible')
+        cy.contains(secondParticipantAnswerText).should('not.exist')
+        cy.contains(facilitatorAnswerText).should('not.exist')
+        cy.contains(leadAnswerText).should('not.exist')
+    })
+
+    const LeadFacilitarorRoles = [Role.Facilitator, Role.OrganizationLead]
```
Comment (achaikou) [inline frontend/cypress/integration/view_answers_spec.ts:73]: For Lead: are leads supposed to see all the answers, or if I am the Lead of PreOps organization I should not be allowed to see all answers to Commissioning questions, but only to PreOps? What about questions in All category?

Regardless of the answer to this - should that be included in out tests? (Doesn't have to be this PR)

File: frontend/cypress/integration/view_answers_spec.ts [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    LeadFacilitarorRoles.forEach(role => {
+        it(`${role} can see answers regardless of role`, () => {
+            const daUser = evalWithIndividualAnswers.participants.find(p => p.role === role)!.user
```
Comment (achaikou) [inline frontend/cypress/integration/view_answers_spec.ts:76]: Remove it? You explicitly call the same in visitProgression

Comment (petterwildhagen) [reply frontend/cypress/integration/view_answers_spec.ts:76]: deleted.

File: frontend/cypress/integration/view_answers_spec.ts (hunk 2 above)
Comment (petterwildhagen) [reply frontend/cypress/integration/view_answers_spec.ts:59]: Fixed.

File: frontend/cypress/integration/view_answers_spec.ts [hunk 5]
```diff
@@ -56,14 +58,12 @@ describe('Viewing answers on stage PREPARATION', () => {
                 text: leadAnswerText,
             })
         )
-    before('Load evaluation with answers by two participants', () => {
+    before('Load evaluation with answers from all two participant, one lead and one facilitator', () => {
```
Comment (achaikou) [inline frontend/cypress/integration/view_answers_spec.ts:61]: from all two participant?

File: frontend/cypress/integration/view_answers_spec.ts (hunk 3 above)
Comment (petterwildhagen) [reply frontend/cypress/integration/view_answers_spec.ts:73]: The organization has no meaning -- it is just a label.

File: frontend/cypress/integration/view_answers_spec.ts (hunk 5 above)
Comment (petterwildhagen) [reply frontend/cypress/integration/view_answers_spec.ts:61]: Fixed

**Review by achaikou — APPROVED** [review]

**Pull request merged**